modules/database/base/utils.py

```
@@ -0,0 +1,59 @@
+        return "String"
+    elif python_type in [int, float]:
+        return "Number"
+    elif python_type == bool:
+        return "True/False"
+    return ""
+
+
+def human_type_to_python_type(human_type: str):
+    try:
+        return float(human_type)  # is it a float?
+    except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)